### PR TITLE
PREL-199 Improving the help output of percona-release

### PR DESCRIPTION
**Pull request by adivinho**
Screenshot 2022-08-10 at 21 47 22

### CODE REVIEW

File: scripts/percona-release.sh [hunk 1]
```diff
@@ -9,9 +9,64 @@ if [[ $(id -u) -gt 0 ]]; then
...
+	    fi
+          fi
+        done
+    done
+  echo -e ${PRODUCTS}
+  echo -e ${OTHER_CURRENT_ALIASES}
+}
+
+function get_repos_from_site {
+  CURRENT_REPOS=$(curl -s https://repo.percona.com | tail -n +28  | grep href | grep -v https | awk -Fhref=\" '{print $2}' | awk -F\/ '{print $1}')
```
Comment (EvgeniyPatlan) [inline scripts/percona-release.sh:49]: what should be in case repo is unavailable?

File: scripts/percona-release.sh [hunk 2]
```diff
@@ -9,9 +9,64 @@ if [[ $(id -u) -gt 0 ]]; then
...
+  ARRAY=$1
+  local flagforabbr="pdmdb"
+  PRODUCTS=()
+  OTHER_CURRENT_ALIASES=()
+  for element in ${ARRAY[@]}
+    do
+      counter=1
+      for abbr in ${PRODUCTS_ABBREVIATIONS[@]}
+        do
+	  if ! [[ "${element//-/}" =~ ^$abbr.*$ ]]; then
```
Comment (EvgeniyPatlan) [inline scripts/percona-release.sh:25]: inverted if. is it possible to rework it?

File: scripts/percona-release.sh [hunk 3]
```diff
@@ -9,9 +9,64 @@ if [[ $(id -u) -gt 0 ]]; then
...
+            if [[ $counter == ${#PRODUCTS_ABBREVIATIONS[@]} ]]; then
+              OTHER_CURRENT_ALIASES+="$element "
+            else
+              counter=$((counter+1))
+            fi
+          else
+            if ! [[ $flagforabbr == $abbr ]]; then
```
Comment (EvgeniyPatlan) [inline scripts/percona-release.sh:32]: inverted if. is it possible to rework it?

File: scripts/percona-release.sh [hunk 4]
```diff
@@ -9,9 +9,64 @@ if [[ $(id -u) -gt 0 ]]; then
...
+          CURRENT_ALIASES+="${repo} "
+      fi
+    done
+
+  CURRENT_REPOS="${CURRENT_REPOS//$'\n'/ }"
+}
+
+get_repos_from_site
+
 ALIASES="ps56 ps57 ps80 psmdb34 psmdb36 psmdb40 psmdb42 pxb24 pxb80 pxc56 pxc57 pxc80 ppg11 ppg11.5 ppg11.6 ppg11.7 ppg11.8 ppg12 ppg12.2 ppg12.3 pdmdb4.2 pdmdb4.2.6 pdmdb4.2.7 pdmdb4.2.8 pdps8.0.19 pdps8.0.20 pdpxc8.0.19 pdps8.0 pdpxc8.0 prel proxysql sysbench pt pmm-client pmm2-client mysql-shell pbm pdmdb4.4 pdmdb4.4.0 psmdb44"
```
Comment (EvgeniyPatlan) [inline scripts/percona-release.sh:65]: looks like it is not needed as on next line you redefine this variable

File: scripts/percona-release.sh [hunk 5]
```diff
@@ -9,9 +9,64 @@ if [[ $(id -u) -gt 0 ]]; then
...
-COMMANDS="enable enable-only setup disable show"
+ALIASES=${CURRENT_ALIASES}
+COMMANDS="enable enable-only setup disable show help"
 REPOSITORIES="original ps-56 ps-57 ps-80 pxc-56 pxc-57 pxc-80 psmdb-36 psmdb-40 psmdb-42 pxb-24 pxb-80 tools ppg-11 ppg-11.5 ppg-11.6 ppg-11.7 ppg-11.8 ppg-12 ppg-12.2 ppg-12.3 pdmdb-4.2 pdmdb-4.2.6 pdmdb-4.2.7 pdmdb-4.2.8 pdps-8.0.19 pdpxc-8.0.19 pdps-8.0.20 pdps-8.0 pdpxc-8.0 prel proxysql sysbench pt mysql-shell pbm pmm-client pmm2-client pdmdb-4.4 pdmdb-4.4.0 psmdb-44"
+REPOSITORIES=${CURRENT_REPOS}
```
Comment (EvgeniyPatlan) [inline scripts/percona-release.sh:69]: looks like it is not needed as on next line you redefine this variable

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: scripts/percona-release.sh (hunk 4 above)
Comment (adivinho) [reply scripts/percona-release.sh]: It has been removed.

File: scripts/percona-release.sh (hunk 5 above)
Comment (adivinho) [reply scripts/percona-release.sh]: Fixed

File: scripts/percona-release.sh (hunk 3 above)
Comment (adivinho) [reply scripts/percona-release.sh]: I have inverted it. Thx.

File: scripts/percona-release.sh (hunk 2 above)
Comment (adivinho) [reply scripts/percona-release.sh]: It has been inverted

File: scripts/percona-release.sh (hunk 1 above)
Comment (adivinho) [reply scripts/percona-release.sh]: I have restored variable REPOSITORIES with default list of repos in such case.

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**